catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/action/AbstractMetacardActionProvider.java

```diff
@@ -30,6 +30,11 @@
 import ddf.action.ActionProvider;
 import ddf.catalog.data.Metacard;
 
+/**
+ * @deprecated Use {@link org.codice.ddf.catalog.actions.AbstractMetacardActionProvider}
```

Comment:
Do we want to add "as of 2.10.0" here so it is easy to see when this was deprecated?